src/css/styles.css

```diff
@@ -194,17 +194,20 @@ body, html {
 }
 
 .moon {
-  height: 6vh;
-  width: 6vw;
+  height: 4em;
```

Comment:
Nice testing out that em sizing. Looking forward to making our site more responsvie